### ISSUE TITLE
Java Script Error before Wallet will Open

### ISSUE THREAD

**Issue by skcheffy** [LN-Zap/zap-desktop#651]
I dont think this is a duplicate of what other errors are logged.

This happens after I enter my Passwork and Sync to Blockchain is catching up.  Picture attached.
![screen shot 2018-08-02 at 8 11 45 am](https://user-images.githubusercontent.com/31999006/43582811-a0564928-962b-11e8-9103-2cf28833d7d1.png)

**Comment by buildfreedom** [comment]
I'm getting exactly the same error using a Mac. Each time I open and sync, shortly after that message is displayed.

**Comment by mrfelton** [comment]
Please can you confirm if the issue exists with the latest code in the master branch @skcheffy @buildfreedom . Thanks.

**Comment by buildfreedom** [comment]
Sorry I’m no coder and cannot work with source, just testing ZAP and giving feedback as asked for. I’ll need to wait until there is an update to the Mac version of the software before I can check.

**Comment by skcheffy** [comment]
Unfortunately i am not a coder also so would not know what to look for.  When there is an update to the GUI I can test then.

Or, if you tell me what to look for i can do that.

**Comment by mrfelton** [comment]
@skcheffy please download the new 0.2.1 release and try with that. Thanks

**Comment by buildfreedom** [comment]
Hi there

Since yesterday I have made repeated attempts to launch this new version, but the wallet always get stuck before completing syncing - currently stuck at 99% - so I haven’t been able to confirm if the previous issue is still present.

FYI, I have quit and restarted the wallet a number of times, but syncing hangs at some point.

For reference, I’m using the MacOS version.

**Comment by mrfelton** [comment]
@buildfreedom how long have you left the wallet syncing for? The last bu=it can take several minutes. The lnd sync process is still a little buggy but we are working on providing some improved feedback to help users understand what is happening in the background.

Please reopen the wallet and leave it for a few minutes. The sync process should finish before long.

**Comment by skcheffy** [comment]
@mrfelton so after re-installing new version mine on the MAC is working fine.  Autopilot is working and channels funded.  Now i also just installed on Linux based computer and followed the install guide.  It sync'ed to about 50% and gave me this message.  Do i need a new thread?


![screenshot from 2018-08-16 18-28-54](https://user-images.githubusercontent.com/31999006/44261980-53306d00-a1e7-11e8-9cba-6b19ede56d52.png)

**Comment by mrfelton** [comment]
@skcheffy please can you test the PR in https://github.com/LN-Zap/zap-desktop/pull/679 which I think should resolve this issue

**Comment by skcheffy** [comment]
Tried the steps and didnt get far and now popped up a new issue and now cant get to part where i would input my password.  See screenshots.  Let me know if I need to move this comment to #679 
![screenshot from 2018-08-17 08-59-07](https://user-images.githubusercontent.com/31999006/44267406-115df180-a1fc-11e8-94c4-b3146f5eb385.png)
![screenshot from 2018-08-17 08-58-06](https://user-images.githubusercontent.com/31999006/44267418-1c188680-a1fc-11e8-8124-1be016ef50af.png)

**Comment by skcheffy** [comment]
I made sure program was shut down completely then tried again.

**Comment by buildfreedom** [comment]
I have been running the sync process now for well over an hour and it’s still stuck at 99%

**Comment by mrfelton** [comment]
I believe all of the issues mentioned in this thread are already resolved in the current master branch. Please could you guys check out the master branch and test with that from a clean slate:

```
rm -r ~/Library/Application\ Support/Electron
git checkout https://github.com/LN-Zap/zap-desktop.git
yarn
yarn dev
```

**Comment by mrfelton** [comment]
I'm going to close this issue do to lack of activity. Please feel free to reopen if the issue persists with the latest release (0.2.2).
